scenarios/platform.benchmarks.yml

```
@@ -39,6 +39,8 @@ scenarios:
   json:
     application:
       job: platformbenchmarks
+      environmentVariables:
+        DOTNET_SYSTEM_NET_SOCKETS_INLINE_COMPLETIONS: "{% if isLinux == true %}1{% else %}0{% endif %}"
```

Nit: Given we're doing the `RuntimeInformation.IsOSPlatform(OSPlatform.Linux)` before we consider setting `UnsafePreferInlineScheduling` anyway, this `isLinux` variable seems redundant.

```suggestion
        DOTNET_SYSTEM_NET_SOCKETS_INLINE_COMPLETIONS: 1
```

---

We don't want to set the environment variable on non-linux machines

---

Why not?

---

Because it makes perf worse. Remember the table we had before:
Sockets Default + Kestrel Default : 1,187,598 RPS
Sockets Inline + Kestrel Inline: 1,242,924 RPS
Sockets Inline + Kestrel Default: 1,072,542 RPS
Sockets Default + Kestrel Inline: 1,153,500 RPS

---

Right, but that was all on linux. Setting this environment variable wouldn't make a difference on Windows.

---

D'oh